backend/routes/stripe.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const router = new express.Router();
+
+const stripe = require('stripe')(process.env.STRIPE_TOKEN);
+
+const SNS_DOMAIN = process.env.REACT_APP_HOST
```

Comment:
@gerardoh13 let’s change line 6 so it’s pulling the REACT_APP_HOST from the config instead of the process. I think that since it’s not setting the process that redirect bug is happening here